src/calendar/model/CalendarModel.ts

```diff
@@ -801,6 +804,17 @@ export class CalendarModel {
 	getFileIdToSkippedCalendarEventUpdates(): Map<Id, CalendarEventUpdate> {
 		return this.fileIdToSkippedCalendarEventUpdates
 	}
+
+	canFullyEditEvent(event: CalendarEvent): boolean {
+		const userController = this.logins.getUserController()
+		const userMailGroup = userController.getUserMailGroupMembership().group
+		const mailboxDetailsArray = this.mailModel.mailboxDetails()
```

Comment:
I don't know if this variable is helpful or confusing

Reply:
I think the fact of mailboxDetails return a list of MailDetails make it confusing tbh. Like, I have to iterate over ```this.mailModel.mailboxDetails()``` to to find the correct ```mailboxDetails```